.github/workflows/sdks.yml

```diff
@@ -480,3 +484,85 @@ jobs:
           api-level: 29
           arch: x86_64
           script: ~/test-toolchain.sh
+      - name: Get cached SDK (aarch64)
+        if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}
+        uses: actions/cache/restore@v4
+        with:
+          path: ~/swift-${{ matrix.version }}-android-aarch64-*-sdk.tar.xz
+          key: ${{ steps.version.outputs.tag }}-aarch64-ndk-27b-sdk
+          fail-on-cache-miss: true
+      - name: Get cached SDK (armv7)
+        if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}
+        uses: actions/cache/restore@v4
+        with:
+          path: ~/swift-${{ matrix.version }}-android-armv7-*-sdk.tar.xz
+          key: ${{ steps.version.outputs.tag }}-armv7-ndk-27b-sdk
+          fail-on-cache-miss: true
+      - name: Assemble SDK
+        if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}
+        id: assemble-sdk
+        run: |
+          set -x
+          ls -lah ~/swift-*-sdk.tar.xz
+
+          # e.g. swift-5.10.1-RELEASE-android-sdk
+          SDK_DIR=${{ steps.version.outputs.tag }}-android-sdk
```

Comment:
Add the API level here and when uploading the SDK: I want that to always be available just when looking at the file, ie `android-$ANDROID_API_LEVEL-sdk`.